Assets/MRTK/SDK/Features/UX/Scripts/InteractiveElement/States/Definitions/CoreInteractionState.cs

```diff
@@ -14,15 +14,24 @@ public enum CoreInteractionState
         Default = 0,
 
         /// <summary>
-        /// Represents the Focus state.
+        /// Represents the Focus state. This state supports both near and far interaction.
         /// </summary>
         Focus,
 
         /// <summary>
-        /// Represents the Touch state.  THIS STATE DOES NOT HAVE LOGIC DEFINED YET.
+        /// Represents the Focus Near state. This is a near interaction state
         /// </summary>
-        Touch
+        FocusNear,
```

Comment:
Will the grabbed / clicked states also be separated as near / far?

Reply:
Yes they will.  

The IMixedRealityPointerHandler will be the entry point for a far interaction select/grab/click/air-tap.  The naming I am thinking of is `FarSelect`

The entry point for a near interaction select is going to be PressableButton's `Pressed` property.  A possible `NearSelect` as the state name.

We could then add new states for any selection entry point that all connect to a `Select` state which will encompass all the selection paths.